src/plugins/veevalidate.js

```diff
@@ -30,7 +32,9 @@ extend('url', {
 localize({
   en,
   es,
-  cn
+  cn,
+  ua,
+  fr
```

Comment:
Same here

Reply:
So you could see the bug if you select French localization and fill a form's field with incorrect value

Reply:
We don’t have French locale in the repo

Reply:
![image](https://user-images.githubusercontent.com/24980019/155740515-f86ae6c1-2cc9-46ae-b0dc-1831853874b2.png)

But I see it is already in master branch

Reply:
Ooops! My bad!